templates/web/base/admin/triage/_inspect.html

```diff
@@ -0,0 +1,69 @@
+[% BLOCK category_list %]
```

Comment:
I see this went away, and then came back, presumably to show the groups? Showing the categories in their groups sounds like a thing the base report-category.html template should probably be doing anyway too?